### PR TITLE
DCS-1040 added feature toggle for new availability check

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: backend/config.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
     url: process.env.BOOK_VIDEO_LINK_UI_URL || `http://localhost:${process.env.PORT || 3000}/`,
     maximumFileUploadSizeInMb: toInt(process.env.MAXIMUM_FILE_UPLOAD_SIZE_IN_MB, 200),
     videoLinkEnabledFor: (process.env.VIDEO_LINK_ENABLED_FOR || '').split(','),
+    newAvailabilityCheckEnabled: (process.env.NEW_AVAILABILITY_CHECK_ENABLED || '').split(','),
```
Comment (andrewrlee) [inline backend/config.js:11]: this needs to be coerced into a boolean rather than a list

Comment (andrewrlee) [reply backend/config.js:11]: `=== 'true'`

Comment (louisenorris) [reply backend/config.js]: Corrected now (whoops)

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**